### PR TITLE
Slå caching af kampagne-beviser fra

### PR DESCRIPTION
**Pull request by kfudbc**
Cache af kampagne-beviser caches ikke længere

### CODE REVIEW

File: src/server/serviceProvider/transformers/Community/Reviews/searchReviews.transform.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import parseReview from '../../../parsers/review.parser';
+import workParser from '../../../parsers/work.parser';
+import {_} from 'lodash';
+
+const SearchReviewsTransform = {
+  event() {
+    return 'searchReviews';
+  },
+
+  requestTransform(event, {}, connection) {
```
Comment (hyatt03) [inline src/server/serviceProvider/transformers/Community/Reviews/searchReviews.transform.js]: MAJOR Change this pattern to not be empty. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3799)

File: src/server/serviceProvider/clients/community.client.js [hunk 2]
```diff
@@ -1430,6 +1430,14 @@ function addGenres(endpoint, {reviewId, genres}) {
   });
 }
 
+function searchReviews(endpoint, {}) {
```
Comment (hyatt03) [inline src/server/serviceProvider/clients/community.client.js]: MAJOR Change this pattern to not be empty. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3799)

**Comment by hyatt03** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by kfudbc** [comment]
hmm, pushed forkert kode, lukker

**Pull request closed without merging**